packages/main/src/themes/sap_horizon/GrowingButton-parameters.css

```diff
@@ -0,0 +1,3 @@
+:root {
```

Comment:
import the base/GrowingButton-parameters.css

Reply:
Done